Source/Core/DotSpatial.Data.Forms/DotSpatial.Data.Forms.csproj

```diff
@@ -12,4 +13,31 @@
     <ProjectReference Include="..\DotSpatial.Symbology\DotSpatial.Symbology.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Compile Update="DataFormsMessageStrings.Designer.cs">
+      <DesignTime>True</DesignTime>
+      <AutoGen>True</AutoGen>
+      <DependentUpon>DataFormsMessageStrings.resx</DependentUpon>
+    </Compile>
+    <Compile Update="DialogImages.Designer.cs">
+      <DesignTime>True</DesignTime>
+      <AutoGen>True</AutoGen>
+      <DependentUpon>DialogImages.resx</DependentUpon>
+    </Compile>
+    <Compile Update="DirectoryView.cs">
+      <Generator>PublicResXFileCodeGenerator</Generator>
+    </Compile>
+  </ItemGroup>
+
+  <ItemGroup>
+    <EmbeddedResource Update="DataFormsMessageStrings.resx">
+      <Generator>ResXFileCodeGenerator</Generator>
+      <LastGenOutput>DataFormsMessageStrings.Designer.cs</LastGenOutput>
+    </EmbeddedResource>
+    <EmbeddedResource Update="DialogImages.resx">
+      <Generator>ResXFileCodeGenerator</Generator>
+      <LastGenOutput>DialogImages.Designer.cs</LastGenOutput>
+    </EmbeddedResource>
+  </ItemGroup>
+
```

Comment:
This lines where added because the .Designer Files are only generated when the .resx file is connected to the ResxFileCodeGenerator, this is something that is not done automatically. And because the ResxFileGenenerator was not connected with with the resx File the old Generated Designer.cs Code didn't match the location and namespaces and didn't find the resources which caused exceptions when Resources where accessed.

Reply:
Ok so we'll leave those. I'm currently wondering why everything compiled ok for me... Can't say I understand Vs 2022 ;)